_spinnaker/configure_kayenta.md

```diff
@@ -249,5 +249,192 @@ environment variable in the manifest, for example), your field might look
+pick up.  Our first real stage is to figure out what's currently running
+in our production Deployment (`Get Baseline`).  For this, we use a
+`Find Artifacts From Resource (Manifest)` stage:
+
+![Get Baseline](https://cl.ly/983f8713210c/Image%202018-09-24%20at%208.58.19%20AM.png)
+
+### Deploy Baseline Manifest
+
+The next two stages (`Deploy Baseline` and `Deploy Canary`) are both `Deploy
+(Manifest)` stages; in this example we're using text manifests for clarity.
+
+Our baseline manifest is named with a `-baseline` suffix to help identify its
+metrics from regular production metrics.  We use a Spinnaker expression to 
+substitute in the current production image hash we found in the previous
+stage, and set an environment variable on these pods to help them differentiate
+their data from other production pods:
+
+```
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: kayentademo-baseline
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: kayentademo
+    template:
+      metadata:
+        labels:
+          app: kayentademo
+      spec:
+        containers:
+          - image: '${#stage(''Get Baseline'').context["artifacts"][0]["reference"]}'
+            name: kayentademo
+            env:
+            - name: version
+              value: baseline
+```
+
+### Deploy Baseline Manifest
+
+Our canary manifest is very much the same, except we've swapped out the
+references to `baseline` (in the deployment name and environment variable)
+and we're using the trigger's build properties to identify the Docker image
+hash:
+
+```
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: kayentademo-canary
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: kayentademo
+    template:
+      metadata:
+        labels:
+          app: kayentademo
+      spec:
+        containers:
+          - image: 'docker.io/armory/kayentademo-testapp:${trigger.properties["TESTAPP_VERSION"]}'
+            name: kayentademo
+            env:
+            - name: version
+              value: canary
+```
+
+### Run Canary Analysis
+
+The Canary Analysis stage is dependent on both of those stages being complete;
+we don't want to begin our analysis until we're sure both the baseline and
+canary deployments are up and running.
+
+We've selected the `DataDog-K8s` Canary Config, a 1-minute delay before
+starting analysis (to give the pods a little more time to settle), and we're
+looking for 5-minute intervals, growing (so the reports look back from the
+start of the run, growing in length over time), and running for a Lifetime
+of 1 hour.
+
+For the Metric Scope, we can simply hardcode the deployment names --
+`kube_deployment:kayentademo-baseline` and `kube_deployment:kayentademo-canary`
+-- since we set them specifically in the previous manifests.  For DataDog,
+the Location fields aren't used, so we've left them blank.
+
+We select our metrics account (Datadog) and our Storage account, and set the
```

Comment:
Should we explain the role of the storage account? It's not immediately obvious that metrics are stored there. Ignore if it's been explained in the same doc.

Reply:
It's explained earlier.  The first half of the doc (which I'd already had merged) goes over the configuration of the feature.  This is just an end-to-end example of using it as a deployment pipeline.